app/models/participant.rb

```diff
@@ -26,10 +28,28 @@ def maneuver_score
     maneuver_participants.sum :score
   end
 
-  def name_and_number
-    if number
-      "#{name} (##{number})"
-    else name
+  def display_information(*options)
+    # option can be any symbol with a corresponding method on Participant.
+    # OR, :bus.
+    result = options.map do |option|
+      case option
+      when :bus
+        option = bus.number
+      when :number
+        if number
+          option = self.send(option).to_s.insert(0, '#')
```

Comment:
Also, you're in a branch where `option` is known to be `:number` so no need for send at all
```ruby
option = "##{number}"
```